### PR TITLE
chore: small refactor

### PR DESCRIPTION
**Pull request by splincode**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

### CODE REVIEW

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/core/components/root/root.style.less [hunk 1]
```diff
@@ -1,5 +1,5 @@
-@import '../../styles/mixins/mixins.less';
-@import '../../styles/mixins/text.less';
+@import '@taiga-ui/core/styles/mixins/mixins';
+@import '@taiga-ui/core/styles/mixins/text';
```
Comment (waterplea) [inline projects/core/components/root/root.style.less:2]: Why not entire local file?

**Review by waterplea — APPROVED** [review]

File: projects/core/components/root/root.style.less (hunk 1 above)
Comment (splincode) [reply projects/core/components/root/root.style.less:2]: what do you mean?

Comment (waterplea) [reply projects/core/components/root/root.style.less:2]: I mean single import of `@taiga-ui/core/styles/taiga-ui-local';`

Comment (splincode) [reply projects/core/components/root/root.style.less:2]: aaa

**Pull request merged**